### PR TITLE
Feature/cram and cleanup

### PR DESCRIPTION
**Pull request by keiranmraine**
Also pulls in an un-merged hotfix branch from Yaobo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,53 @@
...
+- [Usage](#usage)
+- [Dependancies](#dependancies)
+- [Docker, Singularity and Dockstore](#docker-singularity-and-dockstore)
+- [LICENCE](#licence)
+
+<!-- /TOC -->
+
+## Usage
+
+Please see the [wiki](https://travis-ci.org/cancerit/cgpNgsQc/wiki) for details of usage.
```
Comment (drjsanger) [inline README.md]: This isn't the right link. Possibly meant to be [here?](https://github.com/cancerit/cgpNgsQc/wiki) `https://github.com/cancerit/cgpNgsQc/wiki`

**Review by ghost — CHANGES_REQUESTED** [review]
Other than the broken link, looks good.

**Comment by keiranmraine** [comment]
@drjsanger confirm I can copy-paste correctly please

**Review by ghost — APPROVED** [review]

**Comment by drjsanger** [comment]
@keiranmraine looks ok now

**Pull request merged**